src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery extends ColorSupplier {
+    public Ball getRandomBall() {
+
+        Ball ball = new Ball();
+
+        ball.setBallColor(ColorSupplier.getRandomColor());
+
+        ball.setBallNumber(new Random().nextInt(100));
+
+        return ball;
```

Comment:
Delete redundant empty lines.
```suggestion
        Ball ball = new Ball();
        ball.setBallColor(ColorSupplier.getRandomColor());
        ball.setBallNumber(new Random().nextInt(100));
        return ball;
```